### PR TITLE
Feature/about module

### PR DESCRIPTION
**Pull request by JasonLin43212**

### CODE REVIEW

File: src/js/modules/RoutingApp.js [hunk 1]
```diff
@@ -71,6 +85,7 @@ const RoutingApp = ({ sections, roles }) => {
 const mapStateToProps = (state) => ({
   sections: getSections(state),
   roles: getRoles(state),
+  about: getAboutPages(state),
```
Comment (jkao1) [inline src/js/modules/RoutingApp.js]: In `RoutingApp`, rename `about` to `abouts`.

File: src/js/modules/core/components/PageFooter.js [hunk 2]
```diff
@@ -92,14 +102,39 @@ const PageFooter = ({ classes, topLevelSectionsWithDirectChildren }) => {
...
-          The Spectator
+        <Col md={8} mdOffset={2}>
+          <Link to="/" className={classes.theSpectator}>
+            The Spectator
+          </Link>
         </Col>
         <Col md={8} mdOffset={2} className={classes.sectionFlex}>
-          {createLinksToSections()}
+          {linkToSections()}
+          {linkToAbout()}
```
Comment (jkao1) [inline src/js/modules/core/components/PageFooter.js]: Rename these to `createLinksToSections` and `createLinksToAbouts`.

File: src/js/modules/core/components/PageLayout.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ import injectSheet from "react-jss";
 import PageHeader from "./PageHeader";
 import PageFooter from "./PageFooter";
 import { getTopLevelSectionsWithDirectChildren } from "../../sections/selectors";
+import { getAboutPages} from "../../about/selectors";
```
Comment (jkao1) [inline src/js/modules/core/components/PageLayout.js]: Rename to `getAbouts`. It's not the best name, but this way we keep things consistent.

File: src/js/modules/core/components/PageLayout.js [hunk 4]
```diff
@@ -14,21 +15,27 @@ const styles = {
...
-      <PageFooter topLevelSectionsWithDirectChildren={topLevelSectionsWithDirectChildren}/>
+      <PageFooter topLevelSectionsWithDirectChildren={topLevelSectionsWithDirectChildren}
+                  aboutPages={aboutPages}/>
     </div>
   );
 };
 
 const mapStateToProps = (state) => ({
   topLevelSectionsWithDirectChildren: getTopLevelSectionsWithDirectChildren(state),
+  aboutPages: getAboutPages(state),
```
Comment (jkao1) [inline src/js/modules/core/components/PageLayout.js]: Same here—`abouts`.

File: src/js/modules/RoutingApp.js [hunk 5]
```diff
@@ -47,7 +49,18 @@ const RoutingApp = ({ sections, roles }) => {
...
         )}/>
     })
-  }
+  };
+  const createAboutRoutes = () => {
+    return Object.keys(abouts).map(aboutSlug => {
+      const aboutPage = abouts[aboutSlug];
+      return <Route
+        exact path ={`/about/${aboutPage.slug}`}
+        key = {`aboutRoutes${aboutPage.id}`}
```
Comment (jkao1) [inline src/js/modules/RoutingApp.js]: format this code. No spaces around the equals sign.

File: src/js/modules/about/components/AboutPage.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    'fontSize':'18px',
+    'color':'#000000',
+  },
+};
+
+const AboutPage = ({ classes, about }) => {
+  return (
+    <div className={classes.AboutPage}>
+      <p className={classes.aboutTitle}>{about.title}</p>
+      <p className={classes.aboutContent}>{about.content}</p>
```
Comment (jkao1) [inline src/js/modules/about/components/AboutPage.js]: The about content should be in HTML, which means that you need a div with a  `dangerouslySetInnerHTML` attribute (see `ArticleBody.js` for reference).

File: src/js/modules/RoutingApp.js [hunk 7]
```diff
@@ -8,11 +8,13 @@ import { HomePage, PageLayout } from "./core/components";
 import { ArticlePage } from "./articles/components";
 import { SectionPage } from "./sections/components";
 import { RolePage, ContributorPage } from "./users/components";
+import { AboutPage } from "./about/components";
 
 import { getSections } from "./sections/selectors";
 import { getRoles } from "./users/selectors";
+import { getAbouts } from './about/selectors';
```
Comment (NicholasLYang) [inline src/js/modules/RoutingApp.js]: Abouts is not a word

Comment (jkao1) [inline src/js/modules/RoutingApp.js]: yes but it describes the about objects, which carry information about the spec

File: src/js/modules/core/components/PageFooter.js [hunk 8]
```diff
@@ -92,14 +102,39 @@ const PageFooter = ({ classes, topLevelSectionsWithDirectChildren }) => {
       );
     });
   };
+  const createLinksToDescriptions = () => {
+    const descriptionPageLinks = Object.values(descriptionPages)
+      .map(descriptionPage => {
+      return (
```
Comment (jkao1) [inline src/js/modules/core/components/PageFooter.js]: Rename descriptionPage to description.

File: src/js/modules/core/components/PageFooter.js [hunk 9]
```diff
@@ -92,14 +102,39 @@ const PageFooter = ({ classes, topLevelSectionsWithDirectChildren }) => {
       );
     });
   };
+  const createLinksToDescriptions = () => {
```
Comment (jkao1) [inline src/js/modules/core/components/PageFooter.js]: Rename `createLinksToDescriptions` to `createDescriptionLinks`. This has already been done with `createLinksToSections` in a separate module.

File: src/js/modules/core/components/PageFooter.js [hunk 10]
```diff
@@ -92,14 +102,39 @@ const PageFooter = ({ classes, topLevelSectionsWithDirectChildren }) => {
...
+    const descriptionPageLinks = Object.values(descriptionPages)
```
Comment (jkao1) [inline src/js/modules/core/components/PageFooter.js]: Rename `descriptionPageLinks` to `descriptionLinks`.

**Pull request merged**